test/postgresql/index.js

```diff
@@ -0,0 +1,12 @@
+const pg = require("pg");
+
+function main() {
+  new pg.Client().query("select * from customer where id = 1001").catch(() => {
+    /* empty */
```

Comment:
This comment should explain why we're discarding the exceptions, otherwise it can be surprising to the reader.